eclass/linux-mod-r1.eclass

```diff
@@ -303,6 +321,10 @@ fi
 #  3. perform various sanity checks to fail early on issues
 linux-mod-r1_pkg_setup() {
 	debug-print-function ${FUNCNAME[0]} "${@}"
+	# The modules we build are specific to each kernel version, we don't
+	# want to reset the environment to use the user selected kernel version.
+	# Bug 931213, 926063
+	SKIP_KERNEL_BINPKG_ENV_RESET=1
```

Comment:
Can move this after the binary check, it's perserved -- feels kind of awkward shoved at the top.

I'd put it right before `get_version` in `_modules_prepare_kernel()` which houses the linux-info bits